db/bus_test.go

```diff
@@ -201,6 +203,7 @@ var _ = Describe("MessageBus Database Integration", func() {
 					Paused:   true,
 					FixedKey: false,
 					Healthy:  false,
+					Retries: 4,
```

Comment:
Same question, why some are 3 vs 4 vs 2.

Reply:
I'll change this to 3. Some of them are docs or local testing files, none of those are really used.